Makefile

```diff
@@ -70,7 +72,7 @@ endif
 
 .PHONY: test-daemon
 test-daemon: cert build container-env-setup $(CONTAINER_TARGET)
-	@echo "Running the $(PROJECT) in deamon mode..."
+	@echo "Running $(PROJECT) in daemon mode..."
```

Comment:
Thanks for making this fix here, but in the future, it might be a better idea to put this in a separate commit. 😉